docs/bridge/connectors/non_fungible_token_connector copy.mdx

```diff
@@ -3,69 +3,74 @@ title: Non-Fungible Token Connector
 
-Original asset on Calimero means that it was first time ever minted on Calimero, it never existed on NEAR. This means that bridging to NEAR would bring a wrapped NFT on NEAR.
-All steps from the previous two sections apply, its only reversed.
+## Bridging back NFT from Calimero to NEAR (burning the wrapped NFT)
 
-# Automatic bridging
+In blockchain terminology, bridging back NFTs from Calimero to NEAR involves burning/withdrawing wNFTs on Calimero and unlocking NFTs on NEAR.
 
-All of these steps are automated with the Calimero Bridge service. The user/dapp needs only to make a nft_transfer_call when locking NFTs on a chain, the bridge service will do all the `magic` and the wrapped asset will appear on the other chain. Similarly, when someone wants to unlock the funds, they simply need to make a withdraw call on the nft contract, and the bridge service will automatically do its thing and all the components are updated automatically. The token will be unlocked on the other side if token were burnt on the starting chain.
+1. Call `view_mapping` on the Calimero `nft_connector` contract to get the name of the wNFT, as the transfer initiator may only know the NFT name on NEAR.
 
+2. Call `withdraw` on the wNFT (`bridge_token`) on Calimero:
+   - This transaction needs to be proven to demonstrate that it actually occurred.
+   - The method burns the specified token ID of the wNFT on Calimero
 
+3. Call the Calimero RPC for transaction status:
+   - Retrieve the transaction receipt indicating the burn of the specific wNFT with the given token ID and the recipient on the NEAR side.
 
+4. Call the NEAR light client to get the current height:
+   - Ensure that the `current_height` on the NEAR light client is higher than the block height where the wNFTs were withdrawn on the destination chain.
 
+5. Call the Calimero RPC for the light client proof.
+
+6. Call `unlock()` on the NEAR `nft_connector` contract with the obtained proof and a valid block height:
+   - Prove the outcome, and if successful, unlock the NFT on NEAR.
+
+7. Congratulations, the NFT is now unlocked on the source network.
+
+## Bridging between Calimero and NEAR when the original asset is on Calimero
+
+If the original asset exists only on Calimero and not on NEAR, bridging it to NEAR will create a wrapped NFT on NEAR. All the steps from the previous two sections apply but in reverse.
+
+## Automatic bridging
+
+All of these steps are automated with the Calimero Bridge service. Users/DApps only need to make an `nft_transfer_call` when locking NFTs on a chain, and the bridge service will handle the rest, automatically transferring the wrapped asset to the other chain. Similarly, when someone wants to unlock the funds, they simply need to make a `withdraw` call on the NFT contract, and the bridge service will automatically update all the components. If the tokens were burnt on the starting chain, they will be unlocked on the other side.
```

Comment:
same comment as here https://github.com/calimero-is-near/docs/pull/91/files#r1232234770

Reply:
fixed